plugins/Dash/ScopeStyle.qml

```diff
@@ -26,17 +28,22 @@ import Ubuntu.Components 1.3
  */
 
 QtObject {
+
+    property var gsettings: GSettings {
+          schema.id: "com.ubuntu.touch.system-settings"
+      }
+
     /// Style object passed from the scope
     property var style: Object()
 
     /// Color used for text and symbolic icons
-    readonly property color foreground: "foreground-color" in style ? style["foreground-color"] : d.defaultDark
+    property color foreground: gsettings.dashBackground ? "white" : d.defaultDark
```

Comment:
Removed readonly so colors can be changed on gsetting change, if not you have to restart unity8-dash to get the new style changes